README.md

```diff
@@ -116,13 +116,49 @@ The CLI will prompt you to input instructions interactively:
 
 You can configure the demo by specifying the following parameters:
 
-- `--aggregator`: The primary model used for final response generation.
```

Comment:
Why are we renaming this parameter?

Reply:
the documentation is incorrect. Typer will just get the parameters from the main function, where 'model' is used to define the primary model.